lib/project_types/script/layers/infrastructure/languages/rust_task_runner.rb

```diff
@@ -53,6 +58,23 @@ def bytecode
 
             ctx.binread(binary_path)
           end
+
+          def check_tool_version!(tool, min_required_verison)
```

Comment:
I am fine with this for now.  I wonder what's the better long term solution?  Basically, I think CLI shouldn't need to know this much.  All these information about the script project's dependencies should be in the default Script project or in the API toolchain.  I wonder what you think @jacobsteves or @adampetro